streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorRecordContext.java

```diff
@@ -48,10 +48,6 @@ public ProcessorRecordContext(final long timestamp,
         this.headers = headers;
     }
 
-    public void setTimestamp(final long timestamp) {
```

Comment:
This class must be immutable -- otherwise, using `context.forward(..., To.)` in combination with `suppress()` breaks, because `suppress()` buffers a reference to the context and assumes it's immutable.